editor/components/post-taxonomies/flat-term-selector.js

```diff
@@ -68,10 +68,8 @@ class FlatTermSelector extends Component {
 	}
 
 	componentWillUnmount() {
-		this.initRequest.abort();
-		if ( this.searchRequest ) {
-			this.searchRequest.abort();
-		}
+		lodashResult( this.initRequest, 'abort' );
```

Comment:
Struggling to find the difference between `result` and `invoke`.

https://lodash.com/docs/4.17.5#invoke

The latter seems more semantic to what we're doing here (we're not expecting a result) and could avoid the conflict.

Further, as with any use of `get` or Lodash paths in general, we should always define the path as an array, even if only one level deep, as it otherwise triggers Lodash's very underperforming string path parsing. It's not a huge performance concern here, as this won't be called very frequently, but a good habit all the same.

Reply:
Yes invoke, looks more semantic for this case, sorry @mkaz during my suggestion I did not remember the invoke function.

Reply:
Thanks, switched to invoke and didn't realize the performance diff between string and array.